applications/tari_base_node/src/builder.rs

```diff
@@ -217,18 +242,27 @@ pub fn configure_and_initialize_node(
                 id.clone(),
                 peers,
                 &config.peer_db_path,
+                &config.data_dir,
                 db.clone(),
                 mempool,
                 rules.clone(),
+                factories.clone(),
             );
             let outbound_interface = handles.get_handle::<OutboundNodeCommsInterface>().unwrap();
+            let wallet_output_manager_service = handles.get_handle::<OutputManagerHandle>().unwrap();
```

Comment:
There are a lot of unwraps() in here. I would make them Expect's so that if the app crashes the user can get some feedback about what went wrong.